freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -626,6 +626,37 @@ def _create_original_to_rebuilt_nvrs_map(self):
                         and block_build_nvr["version"] == parsed_build_nvr["version"]  # noqa: W503
                     ):
                         nvrs_mapping[block_build] = build_nvr
+
+        if self.event.get("autocomplete_field", False): #TODO implement "autocomplete" field of event
```

Comment:
If this is per event option, we will not be able to enable/disable this for the events triggered by advisories, and can only use this feature while triggering the rebuild by manual. I'd prefer to have a configuration option to enable/disable this (enabled by default), which can be toggle on/off globally.

Reply:
I see! I'll create a parameter in the configurations and fetch it in the `conf` dictionary then.